contracts/contracts/sbtc-withdrawal.clar

```diff
@@ -33,6 +45,110 @@
   )
 )
 
+;; Accept a withdrawal request
+(define-public (accept-withdrawal-request (request-id uint) 
+                                          (bitcoin-txid (buff 32)) 
```

Comment:
As far as I can see, this is only printed right now. Is the intention to verify that this txid exists on-chain in the near future or will we save it for further down the line?

Reply:
On-chain verification using 'clarity-bitcoin' is currently not in the scope due to the limit of 8 inputs & outputs for verification.

Reply:
Great, I know we've talked about this so thanks for repeating it here :+1: